crm_claim_rma/tests/test_claim.py

```diff
@@ -30,3 +30,37 @@ def test_create__with_claim_type(self):
         self.assertEqual(supplier_type, claim.claim_type)
         self.assertIsNotNone(claim.code)
         self.assertTrue(claim.code.startswith('RMA-V/'))
+
+    def test_refund_from_claim(self):
+        invoice = self.env.ref('l10n_generic_coa.demo_invoice_1')
```

Comment:
@naglis can you comment on why these tests work ? I was always lead to believe you can't actually use demo invoice data if you don't depend on the module.

Reply:
@kkarolis why me :slightly_smiling_face: ?

IIUC, the `l10n_generic_coa` addon is [installed implicitly](https://github.com/odoo/odoo/blob/62d7eaf865ca31574b8efb73d3cfb7836ed60da3/addons/account/__init__.py#L28) if no other country `l10n_*` addon is found (based on company country) via a [post-init hook](https://github.com/odoo/odoo/blob/62d7eaf865ca31574b8efb73d3cfb7836ed60da3/addons/account/__manifest__.py#L79) when `account` addon is installed. This is nasty, but depending on `l10n_generic_coa` is not an option to allow the addon to be used with other countries, so I guess there should be a check in the test to skip it if `l10n_generic_coa` is not loaded.

Reply:
I will use other ref not to complicate things.

Reply:
@naglis IIRC we had a small discussion similar to this in some other project, forgive me if i'm wrong :)

Reply:
@adamwojt using other ref is not actually a solution. The problem I had with this is that implicit install and I remember in some tests having issues with that. Anyway, I suggest to leave it as is.

Reply:
@kkarolis I used sale demo ref and created invoice from there.

Reply:
@kkarolis np, but I can't remember. Sorry, if that's the case.